.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -84,32 +66,45 @@ jobs:
         echo "code: $code"
         echo "$code" | grep -q 200
 
+    - name: Set MANUAL_URL
+      id: set-manual-url
+      run: |
+        if [ "${{ github.event_name }}" == "schedule" ]; then
+          echo "MANUAL_URL=https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
+        else
+          echo "MANUAL_URL=https://raw.githubusercontent.com/stackrox/stackrox/${{ github.event.pull_request.head.sha }}/scanner/updater/manual/vulns.yaml" >> "$GITHUB_ENV"
```

Comment:
The value of `github.event.pull_request.head.sha` will likely be an empty string when this job is triggered via `workflow_dispatch`.  

`manual_url` would then end up being `https://raw.githubusercontent.com/stackrox/stackrox//scanner/updater/manual/vulns.yaml` which would yield a `404`

Should handle that scenario IMHO (perhaps by using the current branches ref)